Call pass::show()
Call fail::show()

Fatal error: Uncaught Error: Call to protected method pass::show() from context 'fail' in %s:%d
Stack trace:
#0 %s(%d): fail::show()
#1 {main}
  thrown in %s on line %d